griptape/drivers/image_query/anthropic_image_query_driver.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Optional
+from attr import define, field, Factory
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.drivers import BaseImageQueryDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from anthropic import Anthropic
+
+
+@define
+class AnthropicImageQueryDriver(BaseImageQueryDriver):
+    """
+    Attributes:
+        api_key: Anthropic API key.
+        model: Anthropic model name.
+        client: Custom `Anthropic` client.
+        max_output_tokens: Max output tokens to return.
+    """
+
+    api_key: str = field(kw_only=True, metadata={"serializable": True})
+    model: str = field(default="claude-3-sonnet-20240229", kw_only=True, metadata={"serializable": True})
+    client: Anthropic = field(
+        default=Factory(
+            lambda self: import_optional_dependency("anthropic").Anthropic(api_key=self.api_key), takes_self=True
+        ),
+        kw_only=True,
+    )
+    max_output_tokens: Optional[int] = field(default=4096, kw_only=True, metadata={"serializable": True})
```

Comment:
Do we want to provide a default value here?

Reply:
Anthropic requires *some* value to be specified. I tried with None and it gets angry. So I thought a default would be best because then it's more upfront to the customer vs. hiding it in our internal call. Thoughts?

Reply:
Got it, thanks Anthropic. If we do this, we should probably standardize across the Query Drivers. OpenAi does not have one set, for instance.

Any thoughts @andrewfrench?

Reply:
OpenAI's default no-value behavior is unhelpful enough as to consider it a required parameter as well. Perhaps we should update both the OpenAI and Anthropic drivers to explicitly require this field? In either case, a refactor of the tokenizers + prompt stack to support image and media inputs will go a long way to help here.

Reply:
I'm good with staying consistent across all `image_query` drivers and making `max_output_tokens` a required value for now. That means no default and put the onus on the customer, or should we provide a "middle-of-the-road" default for both OpenAI and Anthropic? I was thinking yes to still providing a default so that customers can use it out-of-the-box without needing to go deeply understand each LLMs tokenization. Thoughts?

Reply:
Put `max_output_tokens` on the base class and made the default 256.

Reply:
Let's keep it named as `max_tokens` for consistency with the Prompt Drivers. We can do a full rename in the future.

Reply:
Done.